package.json

```diff
@@ -1,39 +1,95 @@
 {
-  "name": "fluid-template",
-  "version": "0.1.0",
-  "private": true,
-  "engines": {
-    "node": ">=20.15.1"
-  },
-  "scripts": {
-    "dev": "next dev -p 8080",
-    "build": "next build",
-    "start": "next start -p ${PORT:-8080}",
-    "lint": "next lint"
-  },
-  "dependencies": {
-    "@fontsource/inter": "^5.1.0",
-    "@radix-ui/react-popover": "^1.1.1",
-    "@radix-ui/react-select": "^2.1.1",
-    "@tanstack/react-query": "^5.51.9",
-    "@types/lodash": "^4.17.7",
-    "classnames": "^2.5.1",
-    "lodash": "^4.17.21",
-    "next": "14.2.5",
-    "react": "^18",
-    "react-cookie": "^7.2.2",
-    "react-dom": "^18",
-    "tailwind-merge": "^2.4.0",
-    "zod": "^3.23.8"
-  },
-  "devDependencies": {
-    "@types/node": "^20",
-    "@types/react": "^18",
-    "@types/react-dom": "^18",
-    "eslint": "^8",
-    "eslint-config-next": "14.2.5",
-    "postcss": "^8",
-    "tailwindcss": "^3.4.1",
-    "typescript": "^5"
-  }
+	"name": "reform-fluid-starter",
+	"private": true,
+	"engines": {
+		"node": "22.x"
+	},
+	"scripts": {
```

Comment:
you may also appreciate the updated scripts (they’ll install the submodule automatically after a fresh clone)

you can copy them from the starter and remove typegen